src/v2/view-builder/views/shared/ChallengePushView.js

```diff
@@ -108,12 +113,9 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       return this.options.appState.get('authenticatorKey') === AUTHENTICATOR_KEY.CUSTOM_APP;
     },
 
-    isOVWithAutoChallenge() {
-      return this.isOV() && this.options.appState.getSchemaByName('autoChallenge');
-    },
-
-    isCustomAppWithAutoChallenge(){
-      return this.isCustomApp() && this.options.appState.getSchemaByName('autoChallenge');
+    isAutoChallengeMissingFromRemediation() {
```

Comment:
nit: let's make this a "positive" function, something like `isAutoChallengeSupported`